src/editor/screens/AddActivityCard.js

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import {
+  SafeAreaView,
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  useState
```

Comment:
useState isn't being used currently

Reply:
It will be in the future! Currently it is set up like so:
```
const [searchQuery, setSearchQuery] = React.useState('');
```
So we can change the state of what is written into the search bar, but just no functionality is implemented yet.